src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2017-following-fast-tracks.md

```diff
@@ -572,8 +572,11 @@ If you want to migrate comments to the new storage model, follow the [Comment mi
 To migrate comments to the new storage model:
 
 1. Follow the step from section [Keeping old comments]({{page version='' space='nxdoc' page='upgrade-from-lts-2017-following-fast-tracks'}}#keeping-old-comments).
-2. In the Nuxeo Platform's JSF UI, go to **Admin**&nbsp;> **System Information**&nbsp;> **Migration**, click the button **Migrate comments from relations to the parent id property usage** and wait until migration is completed.
-3. Remove the contribution added at step 1.
+2. In the Nuxeo Platform's JSF UI, go to **Admin**&nbsp;> **System Information**&nbsp;> **Migration**, 
+3. Click the **Detect** button and wait until Nuxeo isolated which comments to migrate,
```

Comment:
The _Detect_ button plays the role of the contribution after the first nuxeo launch.

I think we should split the two approaches, I will push something.

Reply:
In fact, with the new _secured_ implementation present in a HF, we will get rid of the contribution.